README.md

```diff
@@ -1,5 +1,38 @@
 # Matrix
 
+##  TABLE OF CONTENTS
+1. [Installation](#Installation)
+
+2. [Development](##Development)
+
+      [Linux](###Linux)
```

Comment:
Currently, both Linux and Windows hyperlinks appear on the same line. Can you add a blank line to separate them?

Also, please make it an ordered list by adding the respective number in front of it as well as in the Quick Start Guide subsections.

Reply:
Yeah. I'll do.